src/Component/Button/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactNode } from "react";
+import "./button.css";
+export interface buttonProp {
```

Comment:
```suggestion
export interface ButtonProps {
```
The interfaces or types are usually starts with capital letter

Reply:
okay